recipes/hdfs3/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "0.1.1" %}
+
+package:
+  name: hdfs3
```

Comment:
We have `hdfs`, which we name [`python-hdfs`]( https://github.com/conda-forge/python-hdfs-feedstock ). Maybe we should do the same here. Thoughts?